### ISSUE TITLE
bug(ngAria): Use of ng-true-value and ng-false-value not updating aria-checked

### ISSUE THREAD

**Issue by marcysutton** [angular/angular.js#10389]
Checkbox components using `ng-true-value` and `ng-false-value` are not computing the correct `aria-checked` value. 

See this thread from Angular Material: https://github.com/angular/material/pull/882#issuecomment-66353514

Example code:
```
<md-switch ng-model="data.cb2" aria-label="Switch 2" ng-true-value="'yup'" 
ng-false-value="'nope'" role="checkbox" tabindex="0" 
aria-checked="true" aria-invalid="false">
</md-switch>
```

When the checkbox is toggled to the off position using these attributes, the `aria-checked` property is set to `true`. It should be false.
